langchain/document_loaders/trello.py

```diff
@@ -0,0 +1,124 @@
+"""Loader that loads cards from Trello"""
```

Comment:
Do you have any sample data (can be fake data) to help test that the massaging code inside the loader is correct?

Reply:
Thanks for you feedback!
I updated most of it already. Now, about the fake data.... 
what do you need exactly? 
Are you testing loaders (with external Api request) with mockups endpoints?
Asking because I could not find examples in the codebase.
In this case we have 2 abstraction layers the actual trello api and the objects returned by py-trello, that last one is the one we interact directly from langchain.
We could create a Trello "Lang Chain Test" board from a dummy email, since the free plan gives you api access too.
But I guess you will have to keep those safe, langchain maintainers side. 
No sure if that is even possible for your current test setup.